### DISCUSSION TITLE
I want to remove my Gratavar

### DISCUSSION THREAD

**Discussion by toastal** [community/community#29435] [Other Feature Feedback, Questions, & Ideas]
There is a Gravatar associated with an email address I've not had associated with GitHub for years and deleted years ago to. However, GitHub's ID is still tied to this deleted, inaccessible email account for whatever reason. I want the ability to not use Gravatar--especially if I have no matching email accounts to that system. Removing all email addresses from GitHub did nothing.

**Comment by Osoted56** [comment]
Quizás sea la mía por qué recupere mis cuentas

**Comment by radar-blip** [comment]
If you are talking about your identicon, thats tied to your username, not any email addresses. So, as long as you have the same username, you have the same identicon. 

https://github.blog/2013-08-14-identicons/

**Reply by toastal** [reply]
No, I'm talking about removing my association with Gravatar on my account. I would like to revert to whatever the default is, identicon or not. But as it stands, if I remove my profile image, my only option is "Revert to Gravatar" which I have no access to changing and which isn't even associated with any of the emails associated with my GitHub account.
